File: 9-palindrome-number/palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int num=x;
        long long rev=0;
        while(x>0){
            int rem=x%10;
            rev=rev*10+rem;
            x/=10;
        }
        if(num==rev){
            return true;
        }else{
            return false;
        }
    }
};

File: 9-palindrome-number/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
    let num = x,rev=0;
    while (num > 0){
        let rem=num%10;
        rev=rev*10+rem;
        num=Math.floor(num/10)
    }
    if(rev==x){
        return true;
    }
    else{
        return false
        
    }
};